sktm/executable.py

```diff
@@ -205,6 +210,9 @@ def load_config(args):
     else:
         cfg['report_footer'] = os.path.abspath(cfg['report_footer'])
 
+    if not cfg.get('jretry'):
```

Comment:
This condition will never follow through because of the default added to the parser. Because of that, the value will also never be updated by the one from configuration file. I think it would be easier to remove the default from the parser and keep this condition, with an `else` branch that would retype the value to `int` (the value read form configuration file is a string, so leaving that part out will break sktm), what do you say?

Reply:
I misunderstood the logic to load a configuration variable from file. It's updated, pls take a look again.